<launch>
 <!-- Sphere Node-->

  <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="/dev/ttyACM0"/>
	</node>
 
 <!-- sphere node -->
  <node respawn="false" pkg="sphere"
        type="sphere_node" name="sphere" >
  </node>
  
  <!-- mpu node -->
  <node respawn="false" pkg="mpu"
        type="mpu_node" name="mpu" >
  </node>

  <param name="KpVit" value="0.1" type="double"/>
  <param name="KdVit" value="0.01" type="double"/>
  <param name="KiVit" value="0.5" type="double"/>
  
  <param name="KpDir" value="0.1" type="double"/>
  <param name="KdDir" value="0.01" type="double"/>
  <param name="KiDir" value="0.5" type="double"/>
  
</launch>


